### PR TITLE
feat(combobox): reduce change detection cycles

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

**Review by mathisscott — APPROVED** [review]

**Comment by steve-haar** [comment]
GitHub failed to report workflow actions for this PR. Closing and reopening to trigger workflow.

File: projects/angular/src/forms/combobox/combobox.ts [hunk 1]
```diff
@@ -82,23 +82,28 @@ export class ClrCombobox<T>
   extends WrappedFormControl<ClrComboboxContainer>
   implements ControlValueAccessor, LoadingListener, AfterContentInit
 {
-  @ViewChild('textboxInput') textbox: ElementRef;
-  @ViewChild('trigger') trigger: ElementRef;
+  /** The native `<input class="clr-input clr-combobox-input" />` element. */
+  @ViewChild('textboxInput', { static: true }) textbox: ElementRef<HTMLInputElement>;
```
Comment (bbogdanov) [inline projects/angular/src/forms/combobox/combobox.ts:86]: Why we want to run the query before the change detection?

Comment (arturovt) [reply projects/angular/src/forms/combobox/combobox.ts:86]: Queries are always run during the change detection:
```ts
if (rf & ɵRenderFlags.Update) {
  let _t;
  ɵɵqueryRefresh((_t = ɵɵloadQuery())) && (ctx.textbox = _t.first);
}
```

I'm not sure if we have to mark static elements (which are not placed within embedded views) as `{ static: false }`.

**Comment by Jinnie** [comment]
Closing, as due to internal refocusing we can't afford to spend much time on these optimizations.
Thanks a lot for your effort, we may come back to these PRs in future iterations.

**Pull request closed without merging**